### PR TITLE
hdrop: init at 0.2.4

### PR DESCRIPTION
**Pull request by Schweber**
## Description of changes

I messed up #267356 and this is a new PR with a fresh fork. @AndersonTorres please continue with the review and thank you for your patience.

[hdrop](https://github.com/Schweber/hdrop) is a helper script for [Hyprland](https://github.com/hyprwm/Hyprland) window manager that allows the user to run, show and hide specified programs via keybinds.

I am the author of hdrop and would also like to be its maintainer on nixpkgs.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by AndersonTorres — COMMENTED** [review]
LGeTM

File: pkgs/by-name/hd/hdrop/package.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    rev = "v${version}";
+    hash = "sha256-VsM1wPl8edAnZUvYw3IeOHw/XQ2pvbLt0v3G0B8+iSA=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    scdoc
+  ];
+
+  makeFlags = ["PREFIX=$(out)"];
```
Comment (AndersonTorres) [inline pkgs/by-name/hd/hdrop/package.nix:30]:
```suggestion
  makeFlags = [ "PREFIX=$(out)" ];
```

**Comment by Schweber** [comment]
@AndersonTorres I don't understand the ofborg failure. It builds successfully on my machine but not on ofborg. I thought i did something wrong while rebasing your last formatting correction (still struggling a lot with git) but my second push also failed.

Do you understand the reason?

**Comment by AndersonTorres** [comment]
I believe ofborg is in error for another reason besides your code.

Maybe some experimental code arrive on Nixpkgs that broke something.

**Review by AndersonTorres — APPROVED** [review]

File: pkgs/by-name/hd/hdrop/package.nix [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+  lib,
```
Comment (Mindavi) [inline pkgs/by-name/hd/hdrop/package.nix:2]: This stule doesn't seem to match the style used in most packages.

Comment (Schweber) [reply pkgs/by-name/hd/hdrop/package.nix]: I formatted it now with [nixpkgs-fmt](https://nix-community.github.io/nixpkgs-fmt/)

**Review by Mindavi — APPROVED** [review]
Now we wait for OfBorg 🙂

**Comment by Schweber** [comment]
ofborg has no complaints

**Pull request merged**